### PR TITLE
Improvements to scope_upload_classification

### PR DESCRIPTION
**Pull request by bfhealy**
This PR updates scope_upload_classification.py to allow the upload_classification function within to run after import (in addition to being part of an executable script). The PR also adds the ability to skip uploading a period if the value for a given source is missing or NaN. This feature may be useful if the user decides to forego uploading a period associated with a non-periodic label.

### CODE REVIEW

File: tools/scope_upload_classification.py [hunk 1]
```diff
@@ -5,22 +5,36 @@
 from penquins import Kowalski
 from time import sleep
 from scope.fritz import save_newsource, api
+import math
 
 
 def upload_classification(
-    file, gloria, group_ids, taxonomy_id: int, classification: str
+    file: str,
```
Comment (mcoughlin) [inline tools/scope_upload_classification.py:12]: Good on you for starting to type cast, we should be doing that more.

File: tools/scope_upload_classification.py [hunk 2]
```diff
@@ -58,12 +72,17 @@ def upload_classification(
...
                     probs[cls] = row.iloc[-1 * len(classification) + i]
 
-        ra, dec, period = float(row.ra), float(row.dec), float(row.period)
+        ra, dec = float(row.ra), float(row.dec)
+
+        # Check for and assign period
+        period = None  # default
+        if 'period' in columns:
+            period = float(row.period)
+            if math.isnan(period):
```
Comment (mcoughlin) [inline tools/scope_upload_classification.py:81]: Maybe throw a warning if a period is NaN?

**Review by mcoughlin — APPROVED** [review]
Small comments @bfhealy.

File: tools/scope_upload_classification.py (hunk 2 above)
Comment (bfhealy) [reply tools/scope_upload_classification.py:81]: Makes sense, I added separate warnings for NaN and missing period column.

**Pull request merged**